httpd/handler.go

```diff
@@ -485,6 +499,7 @@ func (h *Handler) serveWrite(w http.ResponseWriter, r *http.Request, user *influ
 		writeError(influxdb.Result{Err: err}, http.StatusInternalServerError)
 		return
 	} else {
+		w.WriteHeader(http.StatusOK)
```

Comment:
I don't think we were ever returning 200 properly before, not sure how I didn't catch this.